mlir/tools/mlir-miopen-lib/mlir-miopen-lib.cpp

```diff
@@ -317,7 +317,7 @@ extern "C" MiirStatus miirLowerBin(MiirHandle mlirHandle) {
   pm.addPass(createStripDebugInfoPass());
   pm.addPass(createLowerGpuOpsToROCDLOpsPass(/*indexBitWidth=*/32));
   pm.addPass(createGpuSerializeToHsacoPass(
-      utils.getTriple(), utils.getChip(), utils.getFeatures()));
+      utils.getTriple(), utils.getChip(), utils.getFeatures(), /*optLevel=*/3));
```

Comment:
This pipeline seems to serve for test purpose, do you think we also need an option here for optLevel?

Reply:
Isn't mlir-miopen-lib what gets called when MIOpen calls us? Do we want to give them optLevel control?

Reply:
Ha I think I misread and thought this is `mlir-miopen-driver.cpp`. 

Probably not, since we cant guarantee the correctness on all levels it does more harm than good.

Reply:
Ok, so we'll -O3 the MIOpen people.